fails to build on rawhide: unknown symbol SIGUNUSED

From 144 release: https://koji.fedoraproject.org/koji/taskinfo?taskID=20249286

```
src/common/cockpitunixsignal.c:96:17: error: 'SIGUNUSED' undeclared here (not in a function); did you mean 'SI_USER'?
   { "UNUSED",   SIGUNUSED },    /* 31 (arm,i386,m68k,ppc) */
                 ^~~~~~~~~
                 SI_USER
make[1]: *** [Makefile:4759: src/common/libcockpit_common_a-cockpitunixsignal.o] Error 1
```

--- a/src/common/cockpitunixsignal.c
+++ b/src/common/cockpitunixsignal.c
@@ -92,7 +92,7 @@ struct signv {
   { "LOST",     SIGLOST },      /* 29 (arm,i386,m68k,ppc,sparc*) */
 #endif
   { "PWR",      SIGPWR },       /* 30 (arm,i386,m68k,ppc), 29 (alpha,sparc*), 19 (mips) */
-#ifndef _MIPS_ARCH
+#ifdef SIGUNUSED
   { "UNUSED",   SIGUNUSED },    /* 31 (arm,i386,m68k,ppc) */
 #endif
   { "SYS",      SIGSYS },       /* 31 (mips,alpha,sparc*) */

